docs/en/sql-reference/sql-statements/Administration/SHOW BACKENDS.md

```diff
@@ -37,7 +37,7 @@ Explain:
 4. System Decommissioned is true to indicate that the node is safely offline.
 5. Cluster Decommissioned is true to indicate that the node is rushing downline in the current cluster.
 6. TabletNum represents the number of fragments on the node.
-7. Data Used Capacity represents the space occupied by the actual user data.
+7. Data Used Capacity represents the space occupied by the actual user data, trash data is not included, if you want to get the space used by trash data, please use the show trash statement.
```

Comment:
we should note that showing trash is an io expensive operation, users would better use it when there is little workload.